src/sourmash/cli/utils.py

```diff
@@ -85,6 +85,19 @@ def add_picklist_args(parser):
     )
 
 
+def add_pattern_args(parser):
+    parser.add_argument(
+        '--include-db-pattern',
+        default=None,
+        help='search only signatures that match this pattern in name, filename, or md5'
+    )
+    parser.add_argument(
+        '--exclude-db-pattern',
+        default=None,
+        help='search only signatures that do not match this pattern in name, filename, or md5'
+    )
```

Comment:
do you want to explicitly alias these to `--include` / `--exclude` so that's included in the help? Not sure if everyone is familiar with unique prefix matching...

Reply:
mmmh... good question. will think on it. my hot take for now is to make sure to always refer to them in help etc documentation by their full name, so that if we come up with other include/exclude prefixed options we can use them too.

Reply:
adjusted error message to be clearer in 34f3ac9affff, don't want to alias :)